start-debugger.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Script to help start the users application along with
+# the .NET Core Stackdriver Debugger.
+#
+# Input should be the command to start the users donet
+# application as they normally would.  For example if
+# they would call 'dotnet HelloWorld.dll' they would
+# just run ''./start-debugger.sh dotnet HelloWorld.dll'
+
+start_command=
+while (( "$#" )); do
+  start_command="$start_command $1"
+  shift
+done
+
+# Start the users application to avoid any slow down 
+# while starting the debugger.
+$start_command &
+
+# Start the debugger and attach to the users process.
+APP_PID=$(echo $!)
+# NOTE: This file path is based on logic in package.sh
```

Comment:
Where is package.sh is packaged in the docker image relative to this file?
If we want to produce a docker image that is based off of aspnet core sdk with just this script and agent on top - if that going to be enough of there are other scripts that should be part of that image?

Reply:
package.sh isn't in the docker image. package.sh is just what we use to package things nicely into a zip.